Note: this patch is applied BEFORE 0001-configs-ast-Add-redundnant-env.patch.
There is no way to fit it after, so we should just stick to end of config file
and redefine needed values after the are which is to be patched by said
patch from aspeed-layer.
diff -ru a/include/configs/ast-common.h b/include/configs/ast-common.h
--- a/include/configs/ast-common.h	2021-09-09 00:20:42.514611255 +0300
+++ b/include/configs/ast-common.h	2021-09-09 21:21:54.108111900 +0300
@@ -116,4 +116,20 @@
 	"spi_dma=yes\0" \
 	""
 
+#undef CONFIG_ENV_ADDR
+#undef CONFIG_ENV_ADDR_REDUND
+#undef CONFIG_ENV_OFFSET
+#undef CONFIG_ENV_OFFSET_REDUND
+#undef CONFIG_ENV_SIZE
+#undef CONFIG_ENV_SIZE_REDUND
+#undef CONFIG_BOOTCOMMAND
+
+#define CONFIG_ENV_ADDR		(AST_FMC_CS0_BASE + 0x40000)
+#define CONFIG_ENV_ADDR_REDUND		(AST_FMC_CS0_BASE + 0x50000)
+#define CONFIG_ENV_OFFSET		0x40000
+#define CONFIG_ENV_OFFSET_REDUND	0x50000
+#define CONFIG_ENV_SIZE		0x10000
+#define CONFIG_ENV_SIZE_REDUND		0x10000
+#define CONFIG_BOOTCOMMAND		"bootm 20060000"
+
 #endif	/* __AST_COMMON_CONFIG_H */
